### DISCUSSION TITLE
What is Donald Duck’s car?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#226] [Galaxy Trivia] [answered]
**Hint:** 1934 BR

**Comment by DurandSacha** [comment, marked as answer]
this is a The 313 Convertible !

**Reply by ParanoidUser** [reply]
Almost there! The license plate number of Donald's car is 313, and it is indeed a convertible. Could you guess the car model? 😊

**Reply by DurandSacha** [reply]
It's 1934 Belchfire Runabout ?

**Reply by ParanoidUser** [reply]
> It's 1934 Belchfire Runabout ?

You nailed it! Congratulations on your well deserved [Galaxy Brain](https://github.com/DurandSacha?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 4 answers), and don't hesitate to post your own questions to the community. 😊🎉
